### PR TITLE
TM-4650 (Section One) WS Updates to new pages

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
[Ticket TM-4650](https://metaphase.atlassian.net/browse/TM-4650)

### CODE REVIEW

File: src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx [hunk 1]
```diff
@@ -213,119 +181,96 @@ const ProjectedVacancy = () => {
         <div className="position-search edit-position-details-page">
           <div className="usa-grid-full position-search--header">
             <ProfileSectionTitle title="Projected Vacancy Search" icon="keyboard-o" className="xl-icon" />
-            <div className="results-search-bar">
-              <div className="usa-grid-full search-bar-container">
```
[the remark this answers is not in the document]
Comment (elizabeth-jimenez) [reply src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx:217]: to easily see the actual changes, copy/paste all of this branch's `<div className="results-search-bar pt-20">` content and dev branch's `<div className="results-search-bar">` content into a diff checker

File: src/Constants/Menu.js [hunk 2]
```diff
@@ -170,7 +170,7 @@ export const GET_PROFILE_MENU = () => MenuConfig([
     ],
     children: [
       {
-        text: 'Position Manager',
+        text: 'Position Management',
```
Comment (elizabeth-jimenez) [inline src/Constants/Menu.js:173]: better aligns with our new page names

File: src/Constants/Menu.js [hunk 3]
```diff
@@ -228,7 +228,7 @@ export const GET_PROFILE_MENU = () => MenuConfig([
...
```
Comment (cagbakwu) [inline src/Constants/Menu.js:231]: Good change haha

File: src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx [hunk 4]
```diff
@@ -213,119 +181,96 @@ const ProjectedVacancy = () => {
         <div className="position-search edit-position-details-page">
           <div className="usa-grid-full position-search--header">
             <ProfileSectionTitle title="Projected Vacancy Search" icon="keyboard-o" className="xl-icon" />
```
Comment (jlam-mpc) [inline src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx:183]: This should say "Projected Vacancy Manager" correct? All of the other pages have a header that matches the sidebar title too

File: src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jlam-mpc) [reply src/Components/BureauPage/ProjectedVacancy/projectedVacancy.jsx:217]: Nice cleanup

**Review by jlam-mpc — COMMENTED** [review]
Your changes look good and straightforward! I don't see any logic, visual, or organization issues.

The most minor irrelevant thing not having to do with your ticket is that I noticed the title in the header of Projected Vacancy Management doesn't match the sidebar title like the other features do.

**Comment by elizabeth-jimenez** [comment]
> Your changes look good and straightforward! I don't see any logic, visual, or organization issues.
> 
> The most minor irrelevant thing not having to do with your ticket is that I noticed the title in the header of Projected Vacancy Management doesn't match the sidebar title like the other features do.

Good catch! I went ahead and updated it!

**Review by jlam-mpc — APPROVED** [review]

**Pull request merged**